tests/js/JSONFormatBearTest.py

```diff
@@ -25,6 +26,20 @@
 }"""
 
 
+class ParseExceptionTest(unittest.TestCase):
+
+    def test_parse_err_msg(self):
+        valid_msg = ['Expecting value: line 20 column 30 (char 45)',
+                     'Expecting value: line 1 column 1 ']
+        invalid_msg = ['Expecting property change',
+                       'Expecting line 20 column 30',
+                       'Expecting value: line column']
+        for msg in valid_msg:
+            self.assertTrue(JSONFormatBear.parse_err_msg(msg))
+        for msg in invalid_msg:
+            self.assertFalse(JSONFormatBear.parse_err_msg(msg))
```

Comment:
Could you rather try to assert for the values? See https://github.com/coala/coala-bears/blob/master/tests/vcs/git/GitCommitBearTest.py for how this works^^